test/utils.py

```diff
@@ -615,3 +617,86 @@ def wait_pid_disappear(file, target_pid):
                 found = False
                 break
     assert not found
+
+
+def copy_data(dst, file_paths):
+    for path in file_paths:
+        shutil.copy(path, dst)
+
+
+def get_version(tt_cmd, tmpdir):
+    run_cmd = [tt_cmd, "run", "-v"]
+    instance_process = subprocess.run(
+        run_cmd,
+        cwd=tmpdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+    if instance_process.returncode == 0:
+        stdout = instance_process.stdout
+        full = stdout.splitlines()[0]
+        for word in re.split(r'\s', full):
+            matched = re.match(r'^\d+\.\d+\.\d+', word)
+            if matched:
+                print("Matched:")
+                print(matched)
+                version = re.split(r'\.', matched.group(0))
+                return True, int(version[0]), int(version[1]), int(version[2])
+    return False, 0, 0, 0
+
+
+def is_quit_supported(tt_cmd, tmpdir):
+    ok, major, minor, patch = get_version(tt_cmd, tmpdir)
+    assert ok
+    return major >= 2
+
+
+def is_cluster_app_supported(tt_cmd, tmpdir):
+    ok, major, minor, patch = get_version(tt_cmd, tmpdir)
+    assert ok
+    return major >= 3
+
+
+def is_tuple_format_supported(tt_cmd, tmpdir):
+    ok, major, minor, patch = get_version(tt_cmd, tmpdir)
+    assert ok
+    return major > 3 or (major == 3 and minor >= 2)
+
+
+def is_tarantool_major_one():
```

Comment:
We have `get_tarantool_version()` why we need one more method with similar functionality?
I know you just moved methods here, but since we found the problem, it's better to correct it in the right direction right away.
Please see how it is used in the rest of the tests.

Reply:
Updated.